packages/table/test/selectableTests.tsx

```diff
@@ -31,13 +31,15 @@ describe("DragSelectable", () => {
 
     it("does click selection", () => {
         const onSelection = sinon.spy();
+        const onFocus = sinon.spy();
```

Comment:
do we actually need these spies? considering that not a single one of them seems to be used?

seems to me you could revert this whole file.